src/java/org/apache/cassandra/db/compaction/unified/Controller.java

```diff
@@ -434,8 +434,8 @@ public int RA(long length, int W)
         int bucketIndex = Math.max(0, (int) Math.floor((Math.log(length) - Math.log(m)) / (Math.log(F) - Math.log(o))));
 
         int ret = 0;
-        for (int i = 0; i <= bucketIndex; i++)
-            ret += (W >= 0 ? T - 1 : 1); // W >= 0 => tiered compaction, <0 => leveled compaction
+        for (int i = 0; i < bucketIndex; i++)
```

Comment:
`maxIndex` is inclusive, this should remain `<=`.

Reply:
b0d596ac1d055d975503e55ddcf0b79ddf7bf770

Reply:
Apologies -- it looks like this was an error on my part, the last level is now handled separately and this should indeed be exclusive.